frontend/public/components/overview/_overview.scss

```diff
@@ -1,38 +1,84 @@
+.co-m-nav-title--overview {
```

Comment:
This means we have `.co-m-nav-title--overview` declared in two places.  I recommend consolidating.

Reply:
consolidated rules within overview.scss